Pidaq/Lib/wind_tunnel/air_properties.py

```diff
@@ -0,0 +1,112 @@
+        
+        Parameters:
+            path - path to a CSV file of the air properties
+        """
+        self.logger = logging.getLogger()
+
+        try:
+            open(path) # Open the CSV file
+        except IOError as err:
+            self.logger.error(err)
+            sys.exit()
+        with open(path) as file:
+            csv_reader= csv.reader(file, delimiter =',')
+            next(csv_reader) # Skip the first row
+            tempCol, densityCol, viscosityCol = -1,-1,-1
+            row = next(csv_reader) # Read the second row that contains the units
+        
+            count = 0 # For tracking the column index
+            for cell in row: # This loop determines the column index for 
+                    # temperature, density, and dynamic viscosity based on units
+
+                if cell.find('°C') != -1:
+                    tempCol = count
+
+                if cell.find('[kg/m3]') != -1:
+                    densityCol = count
+
+                if cell.find('[Pa*S]') != -1:
+                    viscosityCol = count
+                count += 1
+
+            # if a unit is not found, terminate the script
+            if tempCol == -1: 
+                self.logger.error('Could not find "°C" in any column header.')
+                sys.exit()
+            if densityCol == -1: 
+                self.logger.error('Could not find "kg/m3" in any column header.')
+                sys.exit()
+            if viscosityCol == -1:
+                self.logger.error('Could not find "Pa*S" in any column header.')
+                sys.exit()
+
+            tempList = []
+            densityList = []
+            viscosityList = []
+            for row in csv_reader: # Populate each list with the relevant values casted to float
+                tempList.append(float(row[tempCol]))
+                densityList.append(float(row[densityCol]))
+                viscosityList.append(float(row[viscosityCol]))
+            
+            # Define the interpolation function for density
+            self.densityFunc = interpolate.interp1d(tempList, densityList,
+                                                    bounds_error = True, assume_sorted = True)
+            
+            # Define the interpolation function for viscosity
+            self.viscosityFunc = interpolate.interp1d(tempList, viscosityList, 
+                                                        bounds_error = True, assume_sorted = True)
+
+    def interpolateDensity(self, inputTemp):
+        """Return interpolated value of density based on temperature.
+        If temperature is out of bounds, return 0.        
+        """
+        try:
+            value= self.densityFunc(inputTemp)
+        except ValueError as e:
+           self.logger.warning("Temperature out of bounds, returning 0. " + str(e))
+           value = 0
+        return value
+    
+    def interpolateViscosity(self, inputTemp):
```

Comment:
Type hinting wouldn't be a bad idea throughout this file

[Reference](https://docs.python.org/3/library/typing.html)

Reply:
It is the first time I also hear about type hinting. I think it is useful, @colton-smith . I added some type hinting. Let me know what you think, and if I missed anything.

Reply:
Since hinting is purely that, just a hint, it doesn't change the way the code works at all, but I think we prefer hinting as much as possible, just makes It a little bit easier from a readability standpoint. Good job here!